src/certFetcher/literal.ts

```diff
@@ -0,0 +1,11 @@
+import { ICertFetcher } from './ICertFetcher.js';
+
+export default class LiteralCertFetcher implements ICertFetcher {
+  // eslint-disable-next-line class-methods-use-this
+  async readCertificate() {
+    return {
+      cert: 'some hard coded cert file',
```

Comment:
do these values get replaced somewhere?

Reply:
No this is just an example how to create a custom fetcher and for testing custom fetcher functionality of the cert fetcher manager.